bazel/dependencies/abseil/0002-remove-maes-and-msse4.1-option-from-cross-compilation.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/absl/copts/GENERATED_AbseilCopts.cmake b/absl/copts/GENERATED_AbseilCopts.cmake
```

Comment:
nothing to do: some of those look suspicious, eg, we don't use cmake, removal of :darwin platforms, etc. But if it works, I'm not going to judge :/ looks good enough, we can refine as needed.

Reply:
Sure @ccontavalli